src/Sylius/Behat/Context/Setup/ProductContext.php

```diff
@@ -931,4 +951,6 @@ private function createChannelPricingForChannel(int $price, ChannelInterface $ch
 
         return $channelPricing;
     }
+
+
```

Comment:
Redundant

Reply:
Both lines empty lines above are still redundant. Please, remove it.